### DISCUSSION TITLE
Remove url from login page in apk

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#666] [General Discussion]
I have installed version 4.6 apk. 
Is there any possible way to remove the url which is displayed on the login page.

![image (1)](https://github.com/Sunbird-Ed/Community/assets/72926048/5b897567-6294-4f9b-8861-37ed27229fcf)

cc: @swayangjit @vpPavithra @luckynvdu @Krishnaj20

**Comment by swayangjit** [comment]
@siddhishinde0723  This needs to be changed in sunbird-mobile-sdk.
**hideurlbar** will hide the adressbar.
https://cordova.apache.org/docs/en/10.x/reference/cordova-plugin-inappbrowser/#:~:text=hideurlbar%3A%20set%20to%20yes%20to,both%20navigation%20buttons%20from%20default.

**Comment by anisha147** [comment]
Hi @siddhishinde0723 

Hope the solution shared by @swayangjit has resolved your query. If yes, could you please close this thread from your side?

cc: @swayangjit [@mohitga](https://github.com/mohitga)
